dashboard/app/views/admin_search/find_students.html.haml

```diff
@@ -53,5 +53,9 @@
         - user.each do |field|
           %td
             %span= field
+        - if user[3]
```

Comment:
:( magic numbers -- is the performance benefit of using pluck rather than loading all fields in the user object worth the readability and maintainability costs of having to use array indexes? (I would guess that the perf benefit is not huge  given the max of 50 users)